src/components/Card/StyledCard.js

```diff
@@ -119,18 +119,18 @@ const CardFooter = ({response, type}) => {
       {metaComponent}
       <MaterialType type={type} />
     </div>
-  ) 
+  )
 }
 CardFooter.propTypes = {
   response: PropTypes.object,
   type: PropTypes.string.isRequired
 }
 
 const StyledCardContainer = styled.div`
-  ${props => cardTypes[props.type]['cardStyles']} 
+  ${props => cardTypes[props.type]['cardStyles']}
   ${props => props.expanded === 'horizontal' ? 'max-width: 760px' : ''}
   ${props => cardTypes[props.type]['shadow']
-    ? `
+  ? `
```

Comment:
I like this indented how it was because I think it's more clear that it's a continuation of the statement the next line up